src/lib/jwt/index.ts

```diff
@@ -17,8 +17,8 @@ async function verifyAndUpdateToken(
 ): Promise<{ response: NextResponse; isAuthenticated: Boolean }> {
   const tokenPayload = (await req.cookies.get("jwtToken")?.value) || "";
 
-  const validToken = /^[\w-]+\.[\w-]+\.[\w-]+$/.test(tokenPayload);
-  if (!tokenPayload || !validToken) {`
+  const validToken = /^[\w-]+\.\.[\w-]+\.[\w-]+\.[\w-]+$/.test(tokenPayload);
```

Comment:
We are using JWT encryption, which differs a little bit from the standard JWT

Reply:
Looks good.